src/main/java/com/example/universe/simulator/entityservice/repositories/PlanetRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.universe.simulator.entityservice.repositories;
+
+import com.example.universe.simulator.entityservice.entities.Planet;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.UUID;
+
+public interface PlanetRepository extends JpaRepository<Planet, UUID> {
```

Comment:
Should have 3 methods: 1. existsByName 2. existsByNameAndIdNot 3. existsByStarId. Method 1 & 2 are used in service add/update operations to check for name parameter uniqueness. Method 3 is used in StarService's delete method to check if the star being deleted is assigned to some planet.